library/std/measurement.qs

```diff
@@ -13,32 +13,39 @@ namespace Microsoft.Quantum.Measurement {
     /// # Description
     /// Measures a register of qubits in the `Z ⊗ Z ⊗ ••• ⊗ Z`
     /// basis, representing the parity of the entire register.
+    /// This operation does not reset the measured qubits to the |0⟩ state,
+    /// leaving them in the state that corresponds to the measurement result.
     ///
     /// # Input
     /// ## register
-    /// The register to be measured.
+    /// The register to be jointly measured.
     ///
     /// # Output
-    /// The result of measuring `Z ⊗ Z ⊗ ••• ⊗ Z`.
+    /// The result of measuring in the `Z ⊗ Z ⊗ ••• ⊗ Z` basis.
     ///
-    /// # Remarks
-    /// This operation does not reset the measured qubits to the |0⟩ state,
-    /// leaving them in the state that corresponds to the measurement result.
+    /// # See also
+    /// - Microsoft.Quantum.Measurement.MeasureEachZ
```

Comment:
I'm glad to have a hint toward `MeasureEachZ` here, but I'm torn on whether it should be a "See Also" or part of remarks so that it can have some explanatory text of why they are related. Something like, "For an operation measures each qubit and returns an array of results, see `Microsoft.Quantum.Measurement.MeasureEachZ`."

Reply:
I added remarks section to MeasureEachZ describing the differences between MeasureEachZ, MeasureAllZ, and MResetEachZ. I also added/kept all 'see also' references.